### PR TITLE
Compose local

### PR DESCRIPTION
**Pull request by davidmeirlevy**

### CODE REVIEW

**Review by itays123 — APPROVED** [review]
Unrelated to the tpoic - do we have the themes ('classic', 'damal' ) documented somewhere?

**Comment by davidmeirlevy** [comment]
we probably should. they are examples of how to create your own customized theme.

**Review by itays123 — COMMENTED** [review]
Provide a file example of the greenpress.local.env file

**Comment by davidmeirlevy** [comment]
@itays123 it should be auto-filled by the Greenpress CLI.
We should mention in docs, that it can retrieve all values of the Greenpress Configuration (I haven't finished describe all properties):
https://docs.greenpress.info/guide/greenpress-configuration.html#the-config-file

File: compose/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "greenpress-docker-scripts",
+  "description": "",
+  "scripts": {
+	"local": "npm run envs && npm run set-local-command && npm run local:compose-up",
+	"local:dev": "npm run envs && npm run set-local-dev-command && npm run local:compose-up",
+	"populate-db": "docker exec greenpress_greenpress_1 npm run populate-db",
+	"stop": "docker rm $(docker ps -f name=greenpress -aq) -f",
```
Comment (or-yam) [inline compose/package.json:8]: on windows this command sends err :
unknown shorthand flag: 'a' in -aq)

**Review by or-yam — COMMENTED** [review]
on windows `stop` command sends err :
```
unknown shorthand flag: 'a' in -aq)
```

File: compose/package.json (hunk 1 above)
Comment (davidmeirlevy) [reply compose/package.json]: I committed a fix. Please check now.

**Pull request merged**